integration/sawtooth_integration/tests/test_poet_liveness.py

```diff
@@ -75,7 +75,7 @@ def test_poet_liveness(self):
 
 def get_block(node):
     try:
-        result = requests.get((URL % node) + "/blocks?count=1")
+        result = requests.get((URL % node) + "/blocks?count=1", timeout=1.0)
```

Comment:
This test will fail at the end if all the validators don't respond within the timeout. One second seems too aggressive and may cause false failures, especially in a resource constrained environment. I almost think that we should just leave it at the full test timeout and make this change for debugging purposes only, but maybe something like 30 seconds is still long enough.